.github/workflows/pr.yml

```diff
@@ -81,6 +81,13 @@ jobs:
           version: v0.11.1
           registry: true
 
+      - name: Start Vault
+        run: .github/install_vault.sh
```

Comment:
I suggest that we use the existing vault.sh instead or creating another one as it will need to maintain 2 files